src/components/ReviewsByCategory.jsx

```diff
@@ -23,7 +23,7 @@ const ReviewsByCategory = () => {
             {reviews.map((review) => {
                 return(
                     <section className="GamesList__GamesCard" key={review.review_id}>
-                        <h3 className="GamesCard__h3">{review.title}</h3>
+                        <Link className="GamesCard__h3" key={review.review_id} to={`/reviews/${review.review_id}`}><h3>{review.title}</h3></Link>
```

Comment:
The reviews by category component is almost identical to the games list component. Can you figure out a way to combine the two. You've done 90% of the work already :)